trashcli/put.py

```diff
@@ -13,6 +13,12 @@
     volume_trash_dir2, my_input
 from .trash import path_of_backup_copy
 from .trash import version
+from . import TRASH_DIRS, PREAMBLE
+
+try:
+    import shtab
+except ImportError:
+    from . import _shtab as shtab
```

Comment:
This code is duplicated. Can you remove the duplication?
Move this logic in `trashcli/shell_completion.py`

Reply:
OK, I'll fix them.